<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>李建樂助理教授實驗團隊簡介</title>
    <meta name="description" content="本網頁簡介李建樂助理教授與其實驗團隊">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+TC&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="../CSS/style1.css">
</head>
<body>
    <nav>
        <ul>
            <li><a href="./HomePage.html">Lab</a></li>
            <li><a href="./Professor.html">Professor</a></li>
            <li><a href="./Member.html">Members</a></li>
            <li><a href="./Thisis.html">Thisis</a></li>
            <li class="logo">logo</li>
        </ul>
    </nav> 
    
    <header>
        <h1 class="headertitle">Profecessor</h1>
    </header>

    <main>
        <div class = information>
            <img src="../Picture/C.Y.Lee.jpg">
            <div>
                <h1>李建樂  助理教授</h1>
                <div class = info>
                    <div>
                        <h2>職務 : 助理教授<br>&emsp;&emsp;&ensp; 行政及諮詢組組長</h2>
                        <h2>辦公室 : 共同科館509室</h2>
                        <h2>E-MAIL : leecy@ntut.edu.tw</h2>
                        <h2>聯絡電話 : 02-27712171#3855</h2>
                        <h2>授課資訊 :</h2>
                        <ul>
                            <li>資料庫</li>
                            <li>計算生物學及醫學資訊</li>
                        </ul>
                    </div>
                    <div>
                        <h2>Research Interests:</h2>
                        <ul>
                            <li>Bioinformatics</li>
                            <li>Artificial Intelligence</li>
                            <li>Computational Biology</li>
                            <li>Big Data Analysis</li>
                            <li>Next Generation Sequencing</li>
                            <li>Database</li>
                            <li>Distributed Computing</li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
        
        <div>
            <h1>EDUCATION</h1>
            <ul>
                <li>Ph.D., 2018, Graduate Institute of Biomedical Electronics and Bioinformatics, National Taiwan University, Taiwan.</li>
                <li>M.S., 2009, Graduate Institute of Biomedical Electronics and Bioinformatics, National Taiwan University, Taiwan.</li>
                <li>B.S., 2007, Electrical Engineering, National Taipei University of Technology, Taiwan.</li>
            </ul>
        </div>
        
        <div>
            <h1>工作經歷</h1>
            <ul>
                <li>
                    Assistant Professor, Master Program in Artificial Intelligence, Innovation Frontier Institute of Research for Science and Technology, NTUT, Taiwan. (Feb. 2024 – Present)
                </li>
                <li>
                    Chief Officer, Administration and Consultation Services Section, Computer and Network Center, NTUT, Taiwan. (Feb. 2024 – Present)
                </li>
                <li>
                    Assistant Professor, Department of Biomedical Engineering, College of Biomedical Engineering, China Medical University, Taiwan. (Aug. 2021 – Feb. 2024)
                </li>
                <li>
                    Assistant Professor, Department of Biomedical Engineering, College of Biomedical Engineering, China Medical University, Taiwan. (Aug. 2021 – Feb. 2024)
                </li>
                <li>
                    Postdoctoral Researcher, Penn Neurodegeneration Genomics Center, Department of Pathology and Laboratory Medicine, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA. (Jan. 2019 – Mar. 2021)
                </li>
                <li>
                    Exchange Visitor, USDA/US Forest Service International Programs, USDA/ARS National Agricultural Library, Maryland, USA. (May 2014 – May 2015)
                </li>
            </ul>
        </div>

        <div>
            <h1>Research Experience</h1>
            
            <h3>Development of a comprehensive pipeline and tool for identifying neoantigens in cancers.(2021 – Present)</h3>
            <ul>
                <li>Proposing a method and analytic pipeline to identify neoantigens from DNA-seq, RNA-seq, and/or liquid chromatography-tandem mass spectrometry data.</li>
                <li>Establishing an online database of tumor neoantigens in the Taiwanese population.</li>
            </ul>
            
            <h3>Development of a deep learning-based method for predicting microRNA targets and an integrated database.(2021 – Present)</h3>
            <ul>
                <li>Development of a novel algorithm for predicting targets by employing the U-Net architecture based on a convolutional neural network.</li>
                <li>Integrating the proposed algorithm with other existing ones as an online platform for providing a reliable predicted result.</li>
            </ul>
            
            <h3>Microbial genome or metagenomics studies.(2021 – Present)</h3>
            <ul>
                <li>Applying a machine learning-based imputation method to improve the current limitations of the Mycobacterium tuberculosis genomic data on transmission inference.</li>
                <li>Prediction of drug resistance genes in non-tuberculosis mycobacterium from whole-genome sequencing data using a genetic algorithm.</li>
                <li>Exploring the associations between gut microbiota and the pediatric diseases of infants in Taiwan.</li>
                <li>Using metagenomic approaches and linear discriminant analysis to identify gut microbiota associated with metabolic syndrome.</li>
            </ul>
            
            <h3>A scalable high-throughput pipeline for inferring the molecular mechanisms of non-coding genetic variants.(2019 – 2021)</h3>
            <ul>
                <li>Development of a post-GWAS tool with scalable, extensible, and distributed computing architecture based on Apache Spark.</li>
                <li>Integration of large-scale functional genomics data for annotation of genetic variants.</li>
                <li>Inferring the molecular mechanisms via tissue contexts, target genes, and downstream biological processes underlying GWAS findings.</li>
            </ul>
            
            <h3>De novo genome assembly and phylogenetic analyses of Mikado and Swinhoe’s pheasants (Syrmaticus mikado and Lophura swinhoii) indigenous to Taiwan.(2014 – 2018)</h3>
            <ul>
                <li>Genome assembly and annotation of Mikado and Swinhoe’s pheasants.</li>
                <li>Identification of gene families and phylogenetic analyses.</li>
                <li>Examination of genes under positive selection and enrichment analysis.</li>
                <li>Identification of the MHC-B region of the Mikado pheasant.</li>
                <li>Investigation of the evolutionary history among long-tailed pheasants.</li>
            </ul>
            
            <h3>Development of web-based online systems for high-throughput data analyses and visualization.(2010 – 2021)</h3>
            <ul>
                <li>MiDSystem: A comprehensive online system for bacterial de novo genome assembly.</li>
                <li>VariED: The first integrated database of gene annotation and expression profiles for variants related to human diseases.</li>
                <li>anamiR: Integrated analysis of microRNA and gene expression profiling.</li>
                <li>CellExpress: A comprehensive microarray-based cancer cell line and clinical sample gene expression analysis online system.</li>
                <li>Development of the i5k Workspace for annotation and visualization of insect genomes.</li>
                <li>miRSystem: An integrated system for characterizing enriched functions and pathways of microRNA targets.</li>
            </ul>
            
            <h3>Propose a prediction algorithm for microRNA targets using a Hidden Markov Model.(2007 – 2009)</h3>
            
            <h3>Development of Cancer Repository System, a web-based management system for digitalization and consolidation of clinical records and specimen data from cancer patients.(2007 – 2008)</h3>
        </div>

        <div>
            <h1>Conference Posters</h1>
            <ul>
                <li>
                    "A CNN-based approach with efficient transfer learning improves microRNA-mRNA prediction", American Association for Cancer Research 2024 Annual Meeting, 聖地牙哥, 2024/04/05 
                </li>
                <li>
                    "TWNeoDB A web-based database for tumor neoantigens in the Taiwanese population", American Association for Cancer Research 2024 Annual Meeting, 聖地牙哥, 2024/04/05 
                </li>
                <li>
                    "A comprehensive online system for identifying tumor neoantigens", American Association for Cancer Research 2023 Annual Meeting, 奧蘭多, 2023/04/14 
                </li>
            </ul>
        </div>
    </main>
</body>
</html>